src/base_client.rs

```diff
@@ -71,9 +71,20 @@ pub struct CurrentRoom {
 }
 
 impl CurrentRoom {
+    // TODO when UserId is isomorphic to &str clean this up.
     pub(crate) fn comes_after(&self, user: &Uid, event: &PresenceEvent) -> bool {
-        if user == &event.sender {
-            event.content.last_active_ago < self.last_active
+        let u = user.to_string();
+        let u = u.split(':').next();
```

Comment:
Why are we now splitting on `:` here? We should compare the full user ids, no?

Reply:
Yes we should. In the test the username in the session object isn't the same as in the sync.json file I'll find the sync file as soon as I get back to my computer.